src/components/screens/login/networkSelector/networkSelector.js

```diff
@@ -127,75 +142,75 @@ const NetworkSelector = ({
-                    <Input
-                      autoComplete="off"
-                      onChange={onChangeInput}
-                      name="customNetwork"
-                      value={address}
-                      placeholder="e.g. https://mainnet-service.lisk.io or 192.168.0.1:4000"
-                      size="xs"
-                      className={`custom-network ${styles.input} ${isValid ? '' : styles.errorInput}`}
-                      onKeyDown={e => e.keyCode === keyCodes.enter && onConnectToCustomNode(e)}
-                      isLoading={isValidationLoading}
-                      status={connected ? 'ok' : 'error'}
-                      feedback={validationError}
-                    />
-                    {
-                      !isValid
-                        ? (
-                          <span className={styles.customNodeError}>
-                            {validationError}
-                          </span>
-                        )
-                        : null
-                    }
-                  </div>
-                  {
-                    isCustomSelected
-                      ? (
-                        <div>
-                          <PrimaryButton
-                            disabled={connected}
-                            /* istanbul ignore next */
-                            onClick={onConnectToCustomNode}
-                            className={`${styles.button} ${styles.backButton} connect-button`}
-                            size="xs"
-                          >
-                            {connected ? t('Connected') : t('Connect')}
-                          </PrimaryButton>
-                        </div>
-                      )
-                      : null
-                  }
-                </span>
-              );
-            }
-
+        networkList.map((network, key) => {
+          if (network.name === networkKeys.customNode) {
             return (
               <span
-                onClick={() => {
-                  onSelectNetwork(network.name);
-                  validateCorrectNode(network.name);
-                  setState({ connected: false });
-                  childRef.current.toggleDropdown(false);
-                }}
+                className={`${styles.networkSpan} address`}
                 key={key}
+                onClick={() => onSelectNetwork(networkKeys.customNode)}
               >
                 {network.label}
+                <div className={styles.inputWrapper}>
+                  <Input
+                    autoComplete="off"
+                    onChange={e => onChangeInput(network.name, e.target.value)}
+                    name="customNetwork"
+                    value={customNetworkUrl}
+                    placeholder="e.g. https://mainnet-service.lisk.io or 192.168.0.1:4000"
+                    size="xs"
+                    className={`custom-network ${styles.input} ${isValid ? '' : styles.errorInput}`}
```

Comment:
@ManuGowda `styles.input` does not correspond to any existing style in the networkSelector.css file, so it can be either fixed, if necessary or better still, removed

Reply:
@ikem-legend good catch, I will check if it can be removed